src/message-box.scss

```diff
@@ -99,8 +100,8 @@ $message-box-states: (
         box-shadow: map_get($state-set, "boxShadow");
       }
 
-      .#{$block}__title {
-        @include fd-icon-base () {
+      .#{$message}__title {
```

Comment:
Why are you bringing `fd-message` component here? It should be the way it was before.

Reply:
Also, we are no longer referencing other components this way. We rely on specificity.

Reply:
fixed